packages/functional-tests/pages/login.ts

```diff
@@ -132,6 +132,14 @@ export class LoginPage extends BaseLayout {
     return this.page.locator(selectors.SUBMIT);
   }
 
+  get signUpCodeHeader() {
```

Comment:
Should this be moved to the confirmSignupCode POM?

Reply:
If it was a simple switch I'd gladly do this, but there are hiccups. For example in: `signinCached.spec.ts > signin cached > unverified cached signin redirects to confirm email`

I suggest to switch over to these POMS in an independent task, this PR is really about removing the unnecessary fixme.